app/models/manageiq/providers/openstack/cloud_manager/vnfd_template.rb

```diff
@@ -0,0 +1,69 @@
+class ManageIQ::Providers::Openstack::CloudManager::VnfdTemplate < ::OrchestrationTemplate
```

Comment:
Move from https://github.com/ManageIQ/manageiq/blob/master/app/models/orchestration_template_vnfd.rb#L1
The original file will be removed after migration is done.

@Ladas Should this class under `CloudManager` or `InfraManager`?